src/mlpack/methods/ann/layer/batch_norm.hpp

```diff
@@ -157,6 +176,12 @@ class BatchNorm
   //! Locally-stored epsilon value.
   double eps;
 
+  //! Locally-stored value for average.
```

Comment:
`If true use average otherwise use momentum for computing running mean and variance over the dataset`
maybe something like this would be better.

Reply:
Sure, I'll make the changes.